app/models/concerns/profile_searching.rb

```diff
@@ -148,6 +156,25 @@ def filter_by_policy(_filter, _operator, policy_or_profile_id)
         conditions: "profiles.id IN (#{any_child_profile.to_sql})"
       }
     end
+
+    def filter_by_policy_and_profile_names(_filter, operator, value)
+      ss_qb = ScopedSearch::QueryBuilder.new(scoped_search_definition, '', scoped_search_definition.profile)
+      values = ss_qb.preprocess_parameters(
+        scoped_search_definition.field_by_name('name'), OPERATORS[operator], value
+      ) do |x|
+        # Passing empty block because nothing needs to happen on yield in preprocess_parameters
+        # The correct yield is invoked in to_ext_method_sql so all notification values are updated correctly
+        # See https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/query_builder.rb
+      end
+
+      policy_cond = sanitize_sql_for_conditions(["policies.name #{operator} (?)", values])
+      profile_cond = sanitize_sql_for_conditions(["profiles.name #{operator} (?)", values])
+
+      profiles = left_outer_joins(:policy).where(policy_cond).or(Profile.where(profile_cond))
+
+      { conditions: 'profiles.id IN (?)',
+        parameter: [profiles.pluck(:id)] }
```

Comment:
Do we really need the `pluck` here? Can't this be written with a simple select instead?

Reply:
It did work with `select` instead of `pluck` 👍